src/collective/upload/profiles/uninstall/cssregistry.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
 <object name="portal_css" meta_type="Stylesheets Registry">
-  <stylesheet remove="True" id="++resource++collective.upload/css/bootstrap.min.css"/>
+  <stylesheet remove="True" id="++resource++collective.js.bootstrap/css/bootstrap.min.css"/>
```

Comment:
@rodfersou you must not remove this here as other add-ons may still be using it.